web/src/features/fireWeather/components/AccuracyColorLegend.tsx

```diff
@@ -44,52 +53,107 @@ const useStyles = makeStyles({
       marginRight: 0
     }
   },
-  green: {
-    backgroundColor: '#07A059',
+  labelContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    width: '260px',
+    'div:nth-child(3n)': {
+      marginRight: 0
+    },
+    justifyContent: 'space-evenly'
+  },
+  darkGreen: {
+    backgroundColor: darkGreenColor,
     borderTop: '2px solid #FFFFFF',
     borderLeft: '2px solid #FFFFFF',
     borderBottom: '2px solid #FFFFFF',
     borderRadius: '2px 0px 0px 2px',
     height: 27,
     width: 32
   },
-  lightGreen: {
-    backgroundColor: '#3BAC48',
+  middleGreen: {
+    backgroundColor: middleGreenColor,
     borderTop: '2px solid #FFFFFF',
     borderBottom: '2px solid #FFFFFF',
     height: 27,
     width: 32
   },
-  lightestGreen: {
-    backgroundColor: '#82C064',
+  lightGreen: {
+    backgroundColor: lightGreenColor,
     borderTop: '2px solid #FFFFFF',
     borderBottom: '2px solid #FFFFFF',
     height: 27,
     width: 32
   },
   neutral: {
-    backgroundColor: '#DFDEDB',
+    backgroundColor: neutralColor,
     borderTop: '2px solid #FFFFFF',
     borderBottom: '2px solid #FFFFFF',
     height: 27,
-    width: 32
+    width: 64
   },
   lightestOrange: {
-    backgroundColor: '#FCCE89',
+    backgroundColor: yellowColor,
     borderTop: '2px solid #FFFFFF',
     borderBottom: '2px solid #FFFFFF',
     height: 27,
     width: 32
   },
-  lightOrange: {
-    backgroundColor: '#F4A036',
+  middleOrange: {
+    backgroundColor: middleOrangeColor,
     borderTop: '2px solid #FFFFFF',
     borderBottom: '2px solid #FFFFFF',
     height: 27,
     width: 32
   },
-  orange: {
-    backgroundColor: '#ED8001',
+  darkOrange: {
+    backgroundColor: darkOrangeColor,
+    borderTop: '2px solid #FFFFFF',
+    borderRight: '2px solid #FFFFFF',
+    borderBottom: '2px solid #FFFFFF',
+    borderRadius: '0px 2px 2px 0px',
+    height: 27,
+    width: 32
+  },
+  darkestRed: {
+    backgroundColor: '#720505',
```

Comment:
I like how the rest of the classes referenced named colors, can we do the same for the remaining ones?